src/components/Auth.js

```diff
@@ -0,0 +1,102 @@
+
+import React, { useState } from "react"
+
+function Auth(props) {
+  let [authMode, setAuthMode] = useState("signin")
```

Comment:
**suggestion:** if there aren't going to be more options other than _signin_ and _signup_, it might be less confusing to use a boolean instead:
`const [..., set...] = useState(true)`